contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -245,76 +247,105 @@ def process_assessment_metadata(ccnode, kolibrinode):
+                    svg_name = "images/{0}.svg".format(image.original_filename)
+                    json_name = "images/{0}-data.json".format(image.original_filename)
+                    if svg_name not in zf.namelist() or json_name not in zf.namelist():
+                        image.file_on_disk.open(mode="rb")
+                        content = image.file_on_disk.read()
+                        content = content.split(exercises.GRAPHIE_DELIMITER)
+                        write_to_zipfile(svg_name, content[0], zf)
+                        write_to_zipfile(json_name, content[1], zf)
+
+            for item in ccnode.assessment_items.all().order_by('order'):
+                write_assessment_item(item, zf)
+
+        finally:
+            zf.close()
+
+def write_to_zipfile(filename, content, zf):
+    info = zipfile.ZipInfo(filename, date_time=(2013, 3, 14, 1, 59, 26))
+    info.comment = "Perseus file generated during export process".encode()
+    info.compress_type = zipfile.ZIP_STORED
+    info.create_system = 0
+    zf.writestr(info, content)
 
 def write_assessment_item(assessment_item, zf):
-    template=''
-    replacement_string = exercises.IMG_PLACEHOLDER + "/images"
-
-    answer_data = json.loads(assessment_item.answers)
-    for answer in answer_data:
-        answer['answer'] = answer['answer'].replace(exercises.CONTENT_STORAGE_PLACEHOLDER, replacement_string)
-
-    hint_data = json.loads(assessment_item.hints)
-    for hint in hint_data:
-        hint['hint'] = hint['hint'].replace(exercises.CONTENT_STORAGE_PLACEHOLDER, replacement_string)
-
-    context = {
-        'question' : assessment_item.question.replace(exercises.CONTENT_STORAGE_PLACEHOLDER, replacement_string),
-        'answers':answer_data,
-        'multipleSelect':assessment_item.type == exercises.MULTIPLE_SELECTION,
-        'raw_data': assessment_item.raw_data.replace(exercises.CONTENT_STORAGE_PLACEHOLDER, replacement_string),
-        'hints': hint_data,
-        'freeresponse':assessment_item.type == exercises.FREE_RESPONSE,
-        'randomize': assessment_item.randomize,
-    }
-
-    if assessment_item.type == exercises.MULTIPLE_SELECTION:
-        template = 'perseus/multiple_selection.json'
-    elif assessment_item.type == exercises.SINGLE_SELECTION:
-        template = 'perseus/multiple_selection.json'
-    elif assessment_item.type == exercises.FREE_RESPONSE:
-        template = 'perseus/input_question.json'
-    elif assessment_item.type == exercises.INPUT_QUESTION:
-        template = 'perseus/input_question.json'
-    elif assessment_item.type == exercises.PERSEUS_QUESTION:
-        template = 'perseus/perseus_question.json'
-
-    result = render_to_string(template, context).encode('utf-8', "ignore")
-    filename = "{0}.json".format(assessment_item.assessment_id)
-    zf.writestr(filename, result)
+    if assessment_item.type != exercises.PERSEUS_QUESTION:
```

Comment:
One idiom I like is making sure that the body of if clauses are small. So in this case, can you add a guard on top:

```
if assessment == PERSEUS_QUESTION:
    return

if assessment == MULTIPLE_SELECTION
```

Notice that we return immediately after the perseus question. We should have the equivalent behaviour as the huge if statement here.